src/apiCalls.js

```diff
@@ -0,0 +1,12 @@
+const getData = (url, object) => {
+    return fetch(url, object)
+      .then(response => {
+        if (!response.ok) {
```

Comment:
I love this catch every time and it's so weird to me it isn't built into fetch. Also though great work on this.